operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml.tmpl

```diff
@@ -212,6 +212,27 @@ spec:
           x-descriptors:
             - urn:alm:descriptor:com.tectonic.ui:label
       version: v1alpha1
+    - description: Represents an installation of latest version of OpenShift Manual Approval Gate
+      displayName: OpenShift Manual Approval Gate
+      kind: OpenShiftManualApprovalGate
+      name: openshiftmanualapprovalgates.operator.tekton.dev
```

Comment:
I see we use `ManualApprovalGate` as kind in CRDs.
But the kind and name looks different here. 
Is there a reason for that?